usaspending_api/agency/tests/integration/test_agency_federal_account_count.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from model_mommy import mommy
+from rest_framework import status
+from usaspending_api.common.helpers.fiscal_year_helpers import current_fiscal_year
+
+url = "/api/v2/agency/{code}/federal_account/count/{filter}"
+
+
+@pytest.fixture
+def agency_data():
```

Comment:
Would using the `agency_account_data` fixture in `conftest.py` work for these tests?

Reply:
no, these test require federal account data

Reply:
you can always inherit one set of data into another and build off of what was there before. I haven't looked into the data enough to know if that will make things simpler or more complex in this case, but it's an option.

Reply:
Looks simple enough to add the `FederalAccount` mock data to conftest, but I was just curious.